README.md

```diff
@@ -5,16 +5,26 @@
 
 Naomi model for subnational HIV estimates
 
+### Installation
+
+The package is not available from [CRAN](https://cran.r-project.org/).
+Instead, the latest version (of this Github repo) may be installed by running the following code from R.
+
+```
+# install.packages("devtools") # Uncomment this line if devtools is not yet installed
+devtools::install_github("mrc-ide/naomi")
+```
+
 ### Development steps
 
 * Make changes in a new branch
 * Run tests from the command line using `make test` or via R using `devtools::test()` to ensure changes don't cause regressions to existing functionality
 * Run build and check before pushing to remote using `make build` and `make check` from command line or `devtools::check()` from R
 * When branch is ready for merging create a PR and add a reviewer
 * Ensure that the version number has been updated according to [semantic versioning](https://semver.org/) and add a news item describing the change
-* Reviewer should check code and ensure the build passes on [travis](https://travis-ci.org/mrc-ide/naomi) before merging
+* Reviewer should check code and ensure the build passes on [Travis](https://travis-ci.org/mrc-ide/naomi) before merging
```

Comment:
```suggestion
* Reviewer should check code and ensure the build passes on [Buildkite](https://buildkite.com/mrc-ide/naomi) before merging
```

Reply:
We're actually using buildkite for this now, would be good to update it whilst we're here